dashboards/service-level.jsonnet

```diff
@@ -163,3 +163,26 @@ stdlib.dashboard(
   .addSeriesOverride({alias: "/.*(SLO Low Threshold)/", color: "#E02F44", linewidth: 3}),
   gridPos={x:12, y: 9, w: 12, h: 9},
 )
+.addPanel(
+  grafana.gaugePanel.new(
+    'SLO targets',
+    min=null,
+    max=null,
+    pluginVersion='7.5.7',
+    reducerFunction='lastNotNull',
+    thresholdsMode='percentage',
+    unit='percentunit'
+  )
+  .addThresholds([
+    { color: 'red', value: 0 },
+    { color: '#EAB839', value: 90 },
+    { color: 'green', value: 99 },
+  ])
+  .addTarget(
+    grafana.prometheus.target(
+      '1- min(slo_target) by (service)',
+      legendFormat='',
+    )
+  ),
+  gridPos={x:0, y: 27, w: 24, h: 5},
```

Comment:
Why is `y: 27`?

Reply:
I just copied the values from Antonia's dashboard, I'm not sure what this actually does.

Reply:
To be honrest I have no idea either. I just created this in the GUI.